js/src/components/paid-ads/asset-group/asset-group-section.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Section from '.~/wcdl/section';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import './asset-group-section.scss';
+
+/**
+ * Renders the form content for managing an asset group of a campaign with Section UI.
+ */
+export default function AssetGroupSection() {
+	return (
+		<Section
+			className=".gla-asset-group-section"
+			title={ createInterpolateElement(
+				__(
+					'Add assets <gray>(Optional)</gray>',
+					'google-listings-and-ads'
+				),
+				{
+					gray: (
+						<span className="gla-asset-group-section__optional-label" />
+					),
```

Comment:
💅 

Maybe we could change this from `gray` to `optional`

Seems more consistent for me and also if the color of the optional label changes in the future it will maintain consistency, whereas gray won't.

Reply:
Thanks for the suggestion. Renamed in https://github.com/woocommerce/google-listings-and-ads/pull/1794/commits/f6e5e9ff668fa1e84c473a93c2e0bf1df2520409